src/food/food.controller.ts

```diff
@@ -24,102 +24,111 @@ export class FoodController {
   }
 
   @Get()
-  @ApiOperation({
-    summary: '음식 리스트를 가져오는 API',
-    description: '사용자 id, category가 주어질 경우 그에 맞는 음식만 가져온다.',
-  })
-  @ApiQuery({
-    name: 'userId, category, hotLevel, sort, size',
-    description: 'userId, category, hotLevel, sort, size',
-    type: FindFoodsQueryDto,
-  })
-  @ApiResponse({ description: '음식 리스트', type: FindFoodDto })
+  @ApiOperation({ summary: '음식 리스트를 가져오는 API' })
   async findFoods(@Query() params: FindFoodsQueryDto): Promise<FindFoodDto[]> {
     return this.foodService.findFoods(params);
   }
 
   @Get('/reviews')
-  @ApiOperation({
-    summary: '리뷰할 음식 리스트를 가져오는 API',
-    description: '리뷰 할 음식 List을 가져온다.',
-  })
-  @ApiCreatedResponse({ description: '음식 list', type: Food })
+  @ApiOperation({ summary: '리뷰할 음식 리스트를 가져오는 API' })
   async reviewFoodList(): Promise<Food> {
-    return this.foodService.findReviewFoods();
+    try {
+      const food: Promise<Food> = this.foodService.findReviewFoods();
+      return food;
+    } catch (e) {
+      throw new HttpException(ERROR_MESSAGE.NOT_FOUND, HttpStatus.BAD_REQUEST);
+    }
   }
 
   @Get('/tests')
-  @ApiOperation({
-    summary: '테스트 음식 리스트를 가져오는 API',
-    description: '테스트에 필요한 음식 리스트를 가져온다.',
-  })
-  @ApiCreatedResponse({ description: '음식 list', type: Food })
+  @ApiOperation({ summary: '테스트 음식 리스트를 가져오는 API' })
   async findTestFoodList(): Promise<Food[]> {
-    return this.foodService.findTestFoods();
+    try {
+      const foods: Promise<Food[]> = this.foodService.findTestFoods();
+      return foods;
+    } catch (e) {
+      throw new HttpException(ERROR_MESSAGE.NOT_FOUND, HttpStatus.BAD_REQUEST);
+    }
   }
 
   @Post()
-  @ApiOperation({
-    summary: '음식 정보를 저장하는 API',
-    description: '새로운 음식 정보를 저장합니다.',
-  })
-  @ApiBody({ type: CreateFoodDto })
-  @ApiCreatedResponse({ description: '등록 된 음식 정보', type: CreateFoodDto })
+  @ApiOperation({ summary: '음식 정보를 저장하는 API' })
   async createFoodInfo(
     @Body() createFoodDto: CreateFoodDto,
   ): Promise<CreateFoodDto> {
-    return this.foodService.createFoodInfo(createFoodDto);
+    const { name, subName, level, category } = createFoodDto;
+    try {
+      const foodLevel = await this.foodService.findFoodLevelByLevelId(level);
+      const foodId = await this.foodService.inputFood(name, subName, foodLevel);
+      const food = this.foodService.setFoodCategory(
+        name,
+        subName,
+        level,
+        category,
+        foodId,
+      );
+
+      return food;
```

Comment:
여기도 굳이 변수할당 하지 않고 바로 return 해도 될 것 같습니다.

```suggestion
      return this.foodService.setFoodCategory(
        name,
        subName,
        level,
        category,
        foodId,
      );
```